### PR TITLE
[SEASK-30] Quick Start -> Automated VM Set Up - Landing Page

### PR DESCRIPTION
**Pull request by raghuramakrishna**

### CODE REVIEW

File: packer/docs/index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```
Comment (jamessmckay) [inline packer/docs/index.html]: please reformat this file, and remove the mixed tabs and spaces. we should be using spaces. did you edit the file in vscode?

File: packer/sea-starter-kit-win2019-eval.pkr.hcl [hunk 2]
```diff
@@ -112,6 +117,7 @@ build {
   provisioner "file" {
     destination = "c:/temp/"
     sources     = [
+		"${path.root}/build/${var.landing_page}.zip",
```
Comment (jamessmckay) [inline packer/sea-starter-kit-win2019-eval.pkr.hcl]: this should be spaces not tabs

File: packer/sea-starter-kit-win2019-eval.pkr.hcl [hunk 3]
```diff
@@ -136,6 +142,59 @@ build {
...
+    ui          = true
+    bubble_text = false
+  }
+
+  provisioner "powershell" {
+    debug_mode        = "${var.debug_mode}"
+    elevated_password = "${var.user_name}"
+    elevated_user     = "${var.password}"
+    inline            = [
+		"((Get-Content -path C:/Ed-Fi/docs/index.html -Raw) -replace '@@DOMAINNAME@@',[System.Net.Dns]::GetHostName()) | Set-Content -Path C:/Ed-Fi/docs/index.html",
```
Comment (jamessmckay) [inline packer/sea-starter-kit-win2019-eval.pkr.hcl]: we should be consistent with the rest of the file and use [Environment]::MachineName

File: packer/sea-starter-kit-win2019-eval.pkr.hcl [hunk 4]
```diff
@@ -136,6 +142,59 @@ build {
...
+        "$Shell = New-Object -ComObject (\"WScript.Shell\")",
+		"$Shortcut = $Shell.CreateShortcut(\"C:/Users/Public/Desktop/SEA Modernization Starter Kit.lnk\")",
```
Comment (jamessmckay) [inline packer/sea-starter-kit-win2019-eval.pkr.hcl]: replace tabs please to spaces

**Review by jamessmckay — CHANGES_REQUESTED** [review]

File: packer/docs/index.html (hunk 1 above)
Comment (raghuramakrishna) [reply packer/docs/index.html]: formatted

File: packer/sea-starter-kit-win2019-eval.pkr.hcl (hunk 2 above)
Comment (raghuramakrishna) [reply packer/sea-starter-kit-win2019-eval.pkr.hcl]: formatted

File: packer/sea-starter-kit-win2019-eval.pkr.hcl (hunk 3 above)
Comment (raghuramakrishna) [reply packer/sea-starter-kit-win2019-eval.pkr.hcl]: updated

File: packer/sea-starter-kit-win2019-eval.pkr.hcl (hunk 4 above)
Comment (raghuramakrishna) [reply packer/sea-starter-kit-win2019-eval.pkr.hcl]: formatted

File: packer/sea-starter-kit-win2019-eval.pkr.hcl [hunk 5]
```diff
@@ -136,6 +142,59 @@ build {
...
+  }
+
+  provisioner "powershell" {
+    debug_mode        = "${var.debug_mode}"
+    elevated_password = "${var.user_name}"
+    elevated_user     = "${var.password}"
+    inline            = [
+        "((Get-Content -path C:/Ed-Fi/docs/index.html -Raw) -replace '@@DOMAINNAME@@',[Environment]::MachineName) | Set-Content -Path C:/Ed-Fi/docs/index.html",
+        "$Shell = New-Object -ComObject (\"WScript.Shell\")",
+		    "$Shortcut = $Shell.CreateShortcut(\"C:/Users/Public/Desktop/SEA Modernization Starter Kit.lnk\")",
```
Comment (jamessmckay) [inline packer/sea-starter-kit-win2019-eval.pkr.hcl]: still tabs in this file

File: packer/docs/index.html [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+        <hr />
+
+        <h4>SharedInstance Assets</h4>
+        <ul>
+            <h3>Suite 3 v5.2.0</h3>
+            <li>
+                API - <a href="https://@@DOMAINNAME@@/WebApi" target="_blank">(Click here)</a>
+                <ul>
+                    <li>Key: dqMi66UoBfLC</li>
+                    <li>Secret: j65JETyk1WNmWKd9tbqMtPUt</li>
```
Comment (jamessmckay) [inline packer/docs/index.html]: this should be generated and/or loaded from a file not checked in

**Review by jamessmckay — CHANGES_REQUESTED** [review]

File: packer/sea-starter-kit-win2019-eval.pkr.hcl (hunk 5 above)
Comment (raghuramakrishna) [reply packer/sea-starter-kit-win2019-eval.pkr.hcl]: i have removed this code

File: packer/docs/index.html (hunk 6 above)
Comment (raghuramakrishna) [reply packer/docs/index.html]: @jamessmckay confirmed with Vinaya on removing the credentials from landing page. I have removed this now

**Review by jamessmckay — APPROVED** [review]

**Review by jamessmckay — CHANGES_REQUESTED** [review]
Build throws an error when run

**Review by jamessmckay — APPROVED** [review]

**Pull request merged**